src/store/poolSlice.ts

```diff
@@ -193,6 +193,12 @@ export const createPoolSlice: StateCreator<
         console.log('error fetching pool data', e);
       }
     },
+    refreshPoolV3Data: async () => {
+      // how to determine which v2 markets to pool? for now always fetch polygon fork
```

Comment:
There's currently no mapping between V2 and V3 markets. An efficient way to handle this might be to add an optional field to `marketsConfig` which is only set for V2 markets and stores an array of V3 markets available to migrate to